crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -322,6 +322,8 @@ def upgrade_first_cluster_node(node, other_node)
         other_node_api = Api::Node.new other_node.name
         node_api.save_node_state("controller")
         save_upgrade_state("Starting the upgrade of node #{node.name}")
+        return false unless evacuate_network_node(node, node["hostname"])
```

Comment:
All services are still up and running and we don't need the ssh step in that case.